src/API/sensors/sensor.controller.js

```diff
@@ -1,10 +1,17 @@
 import Sensor from './sensor.model';
 
-function get(req, res, next) {
-    Sensor.get(req.params.id).then((sensor) => {
-        res.json(sensor);
+/**
+ * Load sensor and append to req.
+ */
+function load(req, res, next, id) {
+    Sensor.get(id).then((sensor) => {
+        req.sensor = sensor;
         return next();
-    }).error(next);
+    }).error((e) => next(e));
```

Comment:
in this very file we have several methods of calling .error
```js
.error((e) => next(e))
.error(next)
```
seem identical, unless error accepts several params